CONTRIBUTING.md

```diff
@@ -98,11 +103,12 @@ the ["fork-and-pull" Git workflow](https://github.com/susam/gitpr)
    CLA).
 2. Fork the repository to your own Github account
 3. Clone the project to your machine
-4. Create a branch locally with a succinct but descriptive name
-5. Commit changes to the branch
-6. Following any formatting and testing guidelines specific to this repo
-7. Push changes to your fork
-8. Open a PR in our repository (to the `develop` branch, **NOT** `main`) and
+4. Checkout the **develop** branch <- `git checkout develop`
+5. Create a branch (again, off of the develop branch) locally with a succinct but descriptive name
+6. Commit changes to the branch
+7. Following any formatting and testing guidelines specific to this repo
```

Comment:
How about we link to these guidelines here to make it clearer what they are?

Reply:
or actually write out the commands you'd type?

Reply:
Not a fan of putting code there (because it might change). Added a few lines to clarify